targets/unity-v2/make.js

```diff
@@ -147,6 +147,7 @@ function makeApi(api, sourceDir, apiOutputDir) {
         generateApiSummary: generateApiSummary,
         getDeprecationAttribute: getDeprecationAttribute,
         getRequestActions: getRequestActions,
+        getCustomApiLogic: getCustomApiLogic,
```

Comment:
I'd like to bring something to your attention regarding this.
In Unity it seems like you already had some logic built for custom behavior, using `getCustomApiFunction`. For C# however, this is not the case. 
However, your unity `getCustomApiFunction` placeholder is inserted _after_ the api call in the Unity case. In our case, we need it to be _before_ the API call.
How does the SDK team feel about if we renamed these two functions to `getPreAPICustomActions` and `getPostAPICustomActions` to keep the naming a bit more consistent here? We could even have the `pre` and `post` be a parameter of a single function like `getApiCustomActions` or something of the sort. 
@pgilmorepf

Reply:
I agree with everything you're saying here, both your methodology and your proposal.

Reply:
Resolving this for now as I realized you use `getCustomApiFunction` **after the whole method** and not **after the request**. My original suggestion was in regards to **pre and post request actions** and so we'll need to perhaps figure out a better name for `getCustomApiLogic` to avoid it from being confused with `getCustomApiFunction`.

Reply:
I agree with everything you're saying here, both your methodology and your proposal.
Come up with good and appropriate names, just make sure you test it.

Reply:
Let's put this in backlog for post GDC then.